README.md

```diff
@@ -2,6 +2,13 @@
 description: Welcome to the home of Adobe Experience Platform Mobile SDK documentation.
 ---
 
+{% hint style="danger" %}
+## The Mobile SDK documentation has moved!
+
+The Adobe Experience Platform Mobile SDK documentation has now moved. The latest version of the documentation can be found at the [Adobe Developer site](https://developer.adobe.com/client-sdks/documentation/) at https://developer.adobe.com/client-sdks/documentation/.
```

Comment:
I suggest we only keep one link for people to click on, for example:
The Adobe Experience Platform Mobile SDK documentation has now moved. The latest version of the documentation can be found at the Adobe Developer site at https://developer.adobe.com/client-sdks/documentation.

Reply:
Yes the double links are unnecessary!

Reply:
@calebk1 Can we also embed an item in the header that links to the new doc location as header is visible of every page.

Reply:
Yeah, the double link was more I wanted to be REALLY SUPER explicit 😂 

But yeah, I can make it just one link.

Reply:
@swarna04 I don't think I can easily do that unless I just go into every doc and paste it on... But I'm not sure if that's the best scenario to do.